tests/integration/fixtures.py

```diff
@@ -1,23 +1,35 @@
 import logging
+import os
 from typing import TYPE_CHECKING
 
+import boto3
+import botocore.config
 import pytest
 
 from localstack.utils.aws import aws_stack
 from localstack.utils.aws.aws_stack import create_dynamodb_table
 from localstack.utils.common import short_uid
 
 if TYPE_CHECKING:
+    from mypy_boto3_cloudformation import CloudFormationClient
     from mypy_boto3_dynamodb import DynamoDBClient
     from mypy_boto3_s3 import S3Client
     from mypy_boto3_sns import SNSClient
     from mypy_boto3_sqs import SQSClient
+    from mypy_boto3_ssm import SSMClient
 
 LOG = logging.getLogger(__name__)
 
 
 def _client(service):
-    return aws_stack.connect_to_service(service)
+    if os.environ.get("TEST_TARGET") == "AWS_CLOUD":
+        return boto3.client(service)
+    config = (
+        botocore.config.Config(connect_timeout=0, read_timeout=0, retries={"total_max_attempts": 1})
+        if os.environ.get("TEST_DISABLE_RETRIES_AND_TIMEOUTS")
```

Comment:
nit: we could use `is_env_true("TEST_DISABLE_RETRIES_AND_TIMEOUTS")` here. (Sorry, missed it in the first round - shouldn't hold back the merge, though...)